src/libostree/ostree-repo-pull.c

```diff
@@ -3392,7 +3392,7 @@ ostree_repo_pull_with_options (OstreeRepo             *self,
       g_autofree char *original_rev = NULL;
 
       if (pull_data->remote_name)
-        remote_ref = g_strdup_printf ("%s/%s", pull_data->remote_name, ref);
+        remote_ref = g_strdup_printf ("%s:%s", pull_data->remote_name, ref);
```

Comment:
Hmm...yes, this looks weird/broken, but this change makes me a bit nervous.  I *think* this code originally came from the `pull-local` code to support `--remote=REMOTE             Add REMOTE to refspec` ?

And the use case for that...hm...I forget.  I think it was something like doing a pull without having to add a `file:///` remote?

Would you mind submitting this patch as a separate PR?